### PR TITLE
Feature/user model

### PR DESCRIPTION
**Pull request by gbrlmarn**

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// define a root route
+app.get('/', (req, res) => {
+    res.send("Hello World");
+});
+
+// Require user routes
+const userRoutes = require('./src/routes/user.routes')
+
+// using as middleware
+app.use('/api/v1/users', userRoutes)
```
Comment (vestemeansorin91) [inline backend/server.js:25]: change '/api/v1/users' to '/api/users'

File: backend/src/controllers/user.controller.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+'use strict';
+const User = require('../models/user.model');
+exports.findAll = function(req, res) {
+User.findAll(function(err, user) {
+  console.log('controller')
```
Comment (vestemeansorin91) [inline backend/src/controllers/user.controller.js:5]: remove console log

File: backend/src/controllers/user.controller.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  if (err)
+  res.send(err);
+  console.log('res', user);
```
Comment (vestemeansorin91) [inline backend/src/controllers/user.controller.js:8]: remove console log

File: backend/src/controllers/user.controller.js (hunk 2 above)
Comment (gbrlmarn) [reply backend/src/controllers/user.controller.js]: am rezolvat

File: backend/src/controllers/user.controller.js (hunk 3 above)
Comment (gbrlmarn) [reply backend/src/controllers/user.controller.js]: ?

**Pull request merged**